docs/usage.md

```diff
@@ -1,6 +1,56 @@
-# nf-core/smrnaseq Usage
-
-## General Nextflow info
+# nf-core/smrnaseq: Usage
+
+## Table of contents
+
+<!-- Install Atom plugin markdown-toc-auto for this ToC to auto-update on save -->
+<!-- TOC START min:2 max:3 link:true asterisk:true update:true -->
+* [Table of contents](#table-of-contents)
+* [Introduction](#introduction)
+* [Running the pipeline](#running-the-pipeline)
+  * [Updating the pipeline](#updating-the-pipeline)
+  * [Reproducibility](#reproducibility)
+* [Main arguments](#main-arguments)
+  * [`-profile`](#-profile)
+  * [`--reads`](#--reads)
+* [Reference genomes](#reference-genomes)
+  * [`--genome` (using iGenomes)](#--genome-using-igenomes)
+  * [`--fasta`](#--fasta)
+  * [`--igenomesIgnore`](#--igenomesignore)
+* [Job resources](#job-resources)
+  * [Automatic resubmission](#automatic-resubmission)
+  * [Custom resource requests](#custom-resource-requests)
+* [AWS Batch specific parameters](#aws-batch-specific-parameters)
+  * [`--awsqueue`](#--awsqueue)
+  * [`--awsregion`](#--awsregion)
+* [Other command line parameters](#other-command-line-parameters)
+  * [`--outdir`](#--outdir)
+  * [`--email`](#--email)
+  * [`-name`](#-name)
+  * [`--seqCenter`](#--seqCenter)
+  * [`-resume`](#-resume)
+  * [`-c`](#-c)
+  * [`--mature`](#--mature)
```

Comment:
It would be good if you put the pipeline specific parameters in the sections above the more generic template parameters. See https://github.com/drpatelh/chipseq/blob/master/docs/usage.md

Reply:
good point. I'll move those  up.